misc/python/materialize/mzcompose/__init__.py

```diff
@@ -246,6 +246,7 @@ def invoke(
                 stdout=stdout,
                 input=stdin,
                 text=True,
+                bufsize=1,
```

Comment:
Do we still need this when we have PYTHONUNBUFFERED=1?

Reply:
That is a good question -- right now, just from pure experimentation, I do not think so, so I planned to remove it. However, from reading the documentation the `bufsize` option has had a tortured history and the default was changed twice I think over the lifetime of the `suprocess` module. So I decided to lock it in place to the value that currently signifies "no buffering".